substrait_consumer/tests/functional/relations/test_sort_relation.py

```diff
@@ -60,6 +73,7 @@ def test_producer_sort_relations(
 
     @custom_parametrization(SORT_RELATION_TESTS)
     @pytest.mark.consume_substrait_snapshot
+    @pytest.mark.usefixtures('mark_consumer_tests_as_xfail')
```

Comment:
You can avoid adding the annotation here if you define the marking fixture as autouse:

https://github.com/voltrondata/spark-substrait-gateway/blob/main/src/gateway/tests/test_dataframe_api.py#L75

Reply:
i only want the fixture to be marked against the consumer tests